src/providers/fault-count/cat-d/fault-count.cat-d.ts

```diff
@@ -170,8 +182,8 @@ export class FaultCountDHelper {
     const result =
       faultTotal +
       sumManoeuvreFaults(manoeuvres, CompetencyOutcome.DF) +
-      pcvDoorExerciseFaultCount +
-      FaultCountDHelper.getVehicleChecksFaultCountCatD(vehicleChecks).drivingFaults;
+      FaultCountDHelper.getVehicleChecksFaultCountNonTrailer(vehicleChecks).drivingFaults +
```

Comment:
should be:

`this.getVehicleChecksFaultCountNonTrailer(vehicleChecks).drivingFaults`